index.js

```diff
@@ -0,0 +1,153 @@
+// Email Input Tag
+const emailInput = document.querySelector(".sign-input");
+// Email Input Container
+const emailInputContainer = document.querySelector(".sign-input-box");
```

Comment:
[수정요청]

@royxk  
불필요한 주석입니다. 변수명으로도 충분히 알 수 있습니다.
코드 전체적으로 주석이 많습니다. 제거 바랍니다.

```suggestion
const emailInput = document.querySelector(".sign-input");
const emailInputContainer = document.querySelector(".sign-input-box");
```